src/control.c

```diff
@@ -13,6 +13,16 @@ static volatile float kd_angular = 1.;
 static volatile int32_t pwm_left;
 static volatile int32_t pwm_right;
 
+static volatile bool collision_detected_signal;
+
+/**
+ * @brief Returns true if a collision was detected.
+ */
+bool collision_detected(void)
+{
+	return collision_detected_signal;
+}
+
 /**
  * @brief Return the current PWM duty for the left motor.
  */
```

Comment:
Update motor_control comment to include collision detection. Maybe include as "error" return of motor control the collision detected?

Reply:
I don't understand this. What do you mean?

Reply:
Sorry, I opened the comment in the wrong place. I mean, the motor_control function docString on lines 109 to 113 from control.c file.

Reply:
Implemented in a0518767ca7279990be1651f5cddbda7e88955a9 ! :blush:

Reply:
good :)